designs/opal/src/back.mjs

```diff
@@ -0,0 +1,765 @@
+  )
+  points.carpenterPocketBottomBack = new Point(
+    points.carpenterPocketTopBack.x,
+    points.carpenterPocketOutseamBottom.y
+  )
+  points.carpenterPocketExtraOutseamTop = points.carpenterPocketOutseamBottom.translate(
+    0,
+    pocketMetric * -options.pocketCarpenterExtraHeight
+  )
+  points.carpenterPocketExtraBackTop = new Point(
+    points.carpenterPocketBottomBack.x,
+    points.carpenterPocketExtraOutseamTop.y
+  )
+  points.carpenterPocketLabel = points.carpenterPocketTopBack
+    .shiftFractionTowards(points.carpenterPocketOutseamBottom, 0.4)
+    .addText('opal:pocketCarpenter', 'center')
+  points.carpenterPocketExtraLabel = points.carpenterPocketBottomBack
+    .shiftFractionTowards(points.carpenterPocketExtraOutseamTop, 1 / 2)
+    .addText('opal:pocketCarpenterExtra', 'center')
+  store.set('carpenterPocketLabel', points.carpenterPocketLabel)
+  store.set('carpenterPocketExtraLabel', points.carpenterPocketExtraLabel)
+
+  // Draft points for the hammer loop.
+  points.hammerLoopTop = points.backPocketFrontBottomP1.shiftFractionTowards(
+    points.backPocketFrontBottomP2,
+    0.5
+  )
+  points.hammerLoopMax = points.hammerLoopTop.translate(
+    pocketMetric * options.hammerLoopCornerX,
+    pocketMetric * options.hammerLoopCornerY
+  )
+  points.hammerLoopOutseam = new Point(
+    0,
+    points.hammerLoopTop.y + pocketMetric * options.hammerLoopOutseam
+  )
+  points.hammerLoopCp1 = points.hammerLoopTop.shiftFractionTowards(
+    points.hammerLoopMax,
+    options.hammerLoopCurve
+  )
+  points.hammerLoopCp2 = points.hammerLoopOutseam.shiftFractionTowards(
+    points.hammerLoopMax,
+    options.hammerLoopCurve
+  )
+  const hammerLoopWidth = pocketMetric * options.hammerLoopWidth
+
+  paths.centerSeam = new Path()
+    .move(points.bibHexagonBottom)
+    .curve(points.bibHexagonBottomCp2, points.cfWaistCp1, points.cfWaist)
+    .line(points.crossSeamCurveStart)
+    .curve(points.crossSeamCurveCp1, points.crossSeamCurveCp2, points.fork)
+    .curve(points.forkCp2, points.inseamHemCp1, points.inseamHem)
+    .addClass('fabric')
+  paths.outseam = new Path().move(points.outseamHem).line(points.waist).addClass('fabric')
+  paths.legHem = new Path().move(points.inseamHem).line(points.outseamHem).addClass('fabric')
+  paths.hem = new Path()
+    .move(points.waist)
+    .curve(points.waistCp2, points.bibHexagonLowerInsideCp1, points.bibHexagonLowerInside)
+    .line(points.bibHexagonUpperInside)
+    .line(points.bibHexagonTop)
+    .curve(points.bibHexagonTopCp2, points.strapTaperInsideCp1, points.strapTaperInside)
+    .line(points.strapEndInside)
+    .line(points.strapEndOutside)
+    .line(points.strapTaperOutside)
+    .curve(
+      points.strapTaperOutsideCp2,
+      points.bibHexagonUpperOutsideCp1,
+      points.bibHexagonUpperOutside
+    )
+    .line(points.bibHexagonLowerOutside)
+    .line(points.bibHexagonBottom)
+    .addClass('fabric')
+  console.log(absoluteOptions.legHemAllowance)
```

Comment:
Please remove this debug output

Reply:
Done

Reply:
I ran npx prettier, and it didn't change any filles:
> npx prettier --write 'src/*.mjs' 'tests/*.mjs'

src/back.mjs 229ms (unchanged)
src/bib.mjs 80ms (unchanged)
src/bibplacket.mjs 30ms (unchanged)
src/front.mjs 71ms (unchanged)
src/hammerloop.mjs 19ms (unchanged)
src/index.mjs 5ms (unchanged)
src/pocketback.mjs 10ms (unchanged)
src/pocketbib.mjs 13ms (unchanged)
src/pocketcarpenter.mjs 10ms (unchanged)
src/pocketcarpenterextra.mjs 7ms (unchanged)
src/pocketslash.mjs 18ms (unchanged)
src/pocketslashshield.mjs 10ms (unchanged)
src/waistband.mjs 22ms (unchanged)
tests/shared.test.mjs 2ms (unchanged)